### PR TITLE
survey results

### PR DESCRIPTION
**Pull request by nevrome**
My idea was to simply include the main points made in the survey into the statement of purpose. No extra evaluation or quantitative analysis of the results. It was exploratory anyway and I don't think it's necessary to focus to much on it.

Maybe you, @MartinHinz, go through the results once more, just to make sure I didn't miss something important. 

Once the statement text is ready from our point of view I suggest to write to the 30 people who participated in the survey to ask for their feedback.

### CODE REVIEW

File: statement.Rmd [hunk 1]
```diff
@@ -2,28 +2,50 @@
...
 
-The Special Interest Group is an open format that is intended to support discussion, the circulation of new ideas, and the integration of different approaches in connection with the objectives. Everyone may participate on equal terms in accordance with the Code of Conduct, as is usual in open source software development.
+The Special Interest Group is an open format that is intended to support discussion, the circulation of new ideas, and the integration of different approaches in connection with the objectives. Everyone may participate on equal terms in accordance with the [Code of Conduct](...), as is usual in open source software development. To be as inclusive as possible and also for the sake of reproducibility we strongly prefer open source over proprietary software.
```
Comment (MartinHinz) [inline statement.Rmd]: before making this available we should than have a code of conduct!

File: statement.Rmd [hunk 2]
```diff
@@ -2,28 +2,50 @@
...
+The Special Interest Group is an open format that is intended to support discussion, the circulation of new ideas, and the integration of different approaches in connection with the objectives. Everyone may participate on equal terms in accordance with the [Code of Conduct](...), as is usual in open source software development. To be as inclusive as possible and also for the sake of reproducibility we strongly prefer open source over proprietary software. 
+
+We especially invite students and young career researches to become part of a wider network of developers and users to improve their skills, and promote the use of scripting languages in research, publication and teaching in archaeology. Open and reproducible research is becoming increasingly important in all scientific disciplines. We have to avoid a "lost" generation of students in archaeology that can't keep pace with this development due to a lack of hard knowledge about computational tools and data exchange standards. The SIG aims to establish the career path of the research software engineer in archaeology.
```
Comment (MartinHinz) [inline statement.Rmd]: Fine with that

File: statement.Rmd [hunk 3]
```diff
@@ -2,28 +2,50 @@
...
 
 ### Scripting languages
 
 A scripting language is a programming language which allows live coding interaction with an software interpreter to perform tasks with data. It does not require the compilation of a software module (or only requires this incidentally to the main coding activity) and it is therefore linked to a rapid development style especially suitable for scientific data exploration, analysis, and visualisation. The following list includes popular scripting languages in archaeology and many other sciences:
 
 - [R](https://www.r-project.org/) 
 - [Python](https://www.python.org/)
+- [Bash](http://www.gnu.org/software/bash/)
```
Comment (MartinHinz) [inline statement.Rmd]: bash? Really?

File: statement.Rmd [hunk 4]
```diff
@@ -2,28 +2,50 @@
...
 - [Netlogo](https://ccl.northwestern.edu/netlogo/)
 - [Stan](http://mc-stan.org/)
 - [OxCal](https://c14.arch.ox.ac.uk/oxcal.html)
 - ...
 
-The scopes and purposes of these scripting languages are highly different. Some of them are fully capable programming languages which provide methods and libraries for all kind of cross-platform system- and network-interaction. Some are rather specific collections of basic commands to control individual software tools and therefor serve as mere interfaces.
+The scopes and purposes of these scripting languages are highly different. Some of them (like R or Python) are fully capable programming languages which provide methods and libraries for all kind of cross-platform system- and network-interaction. Some (like OxCal) are rather specific collections of basic commands to control individual software tools and therefore serve as mere interfaces. Despite this differences both kind of scripting languages allow to create a reproducible representation of a scientific task.
```
Comment (MartinHinz) [inline statement.Rmd]: great

File: statement.Rmd [hunk 5]
```diff
@@ -2,28 +2,50 @@
...
+The use of scripting languages for research allows the user to document every necessary step in a research pipeline. The resulting script can be used to reproduce the data analysis by other researchers, given that also the input datasets are provided and -- if necessary -- the required software environment can be emulated. Scripting languages are therefore the ideal for reproducible research. R and and Python for example provide specialised frameworks to combine data, code and text (R Markdown, Jupyter Notebooks, ...). Reproducible research is important for archaeology because it allows other researchers to verify published findings, and to easily extend new methods to other datasets.
```
Comment (MartinHinz) [inline statement.Rmd]: fine

File: statement.Rmd [hunk 6]
```diff
@@ -2,28 +2,50 @@
...
 
 ### Version control
 
 Reproducible research ideally also documents the science production process with all inductive and deductive steps of hypothesis building. A modern and powerful way to store a commented succession of changes in text and scripted data analysis is available with tools like Git or SVN which where initially designed for software development. They provide a simple environment to keep track of even minor changes, go back to earlier working stages and allow for well documented collaborations thanks to forking. The combination of scripting languages and version control is an important basis for a fully comprehensible research process.
 
+### Objectives of the SIG-SSLA
```
Comment (MartinHinz) [inline statement.Rmd]: High expectations, but why not. Might be altered in the course of discussion and by reality later on

File: statement.Rmd (hunk 3 above)
Comment (nevrome) [reply statement.Rmd]: [Yess!](https://stackoverflow.com/questions/28693737/is-bash-a-programming-language) I would say it's even a pretty common language in data analysis pipelines. None of our projects works without some shell scripting.

File: statement.Rmd (hunk 6 above)
Comment (nevrome) [reply statement.Rmd]: Ja - let's wait for the feedback.

**Comment by nevrome** [comment]
Creating a code of conduct is a pretty big deal. Fortunately we don't have to care about the really difficult stuff: The [Ethics Policy](http://caa-international.org/about/ethics-policy/) of the CAA international already covers this. I'm just thinking about adding some lines that fit better for software development and sharing code on github. Maybe we create a new tab for the code of conduct, link to the Ethics Policy and add a a modified version of [this](https://www.contributor-covenant.org/version/1/0/0/code-of-conduct.html).

**Pull request merged**